src/walletConnect/screens/RequestContent.tsx

```diff
@@ -177,6 +191,16 @@ const styles = StyleSheet.create({
   closeButton: {
     alignSelf: 'flex-end',
   },
+  placeholderLogoBackground: {
+    backgroundColor: Colors.light,
+    marginRight: -Spacing.Small12,
+    borderColor: Colors.gray2,
+    borderWidth: 1,
+  },
+  placeholderLogoText: {
+    ...fontStyles.h2,
```

Comment:
I saw the text is not vertically centered, I think setting `lineHeight: undefined` helps with this.

Reply:
ah yes, thank you for your pixel perfect vision! i didn't even notice 🙈

removing the line height worked, i also changed the size to h1 to make the text size match the valora icon more
![Simulator Screen Shot - iPhone 11 - 2022-06-24 at 10 41 48](https://user-images.githubusercontent.com/20150449/175498690-1906489e-60c4-4a77-ad97-e11b9fa51e76.png)